### PR TITLE
Change Highest to Most Voted

### PR DESCRIPTION
**Pull request by felipecrs**
I think that's what it refers. Am I right? We had our first battle in our team using Thunderdome and we collected this feedback.

### CODE REVIEW

File: frontend/public/lang/de.json [hunk 1]
```diff
@@ -275,7 +275,7 @@
       "voteResults": {
         "totalVotes": "Sch\u00E4tzungen",
         "average": "Durchschnitt",
-        "highest": "H\u00F6chster",
+        "highest": "Meistgew\u00E4hlt",
```
Comment (felipecrs) [inline frontend/public/lang/de.json:278]: PS: Google Translate

**Comment by StevenWeathers** [comment]
So technically "highest" is a weighted most voted, e.g. if both `2` and `5` had the same vote counts `5` is the highest number so that is what is displayed.  I'm open to changing it to "most voted" but I think there needs to be a way to signal that its a weighted result (which Highest implies).

**Comment by felipecrs** [comment]
Got it. Some options are:

- Highest Most Voted

- ⬆️ Most Voted

- Most Voted

File: frontend/public/lang/de.json (hunk 1 above)
Comment (Plunts) [reply frontend/public/lang/de.json:278]: Google is correct ;)

**Comment by StevenWeathers** [comment]
Closing as the verbiage is sufficient for now.

**Pull request closed without merging**